server/api/controllers/audit/auditController.js

```diff
@@ -19,10 +19,12 @@ let url = require('url');
 
 function createAuditLogQuery(since, until, exclusiveStartKey, perPage, filter) {
   let rq = {
-    limit: perPage || 10,
     maxDate: until.toString(),
     minDate: since.toString()
   };
+  if (perPage) {
+    rq.limit = perPage || 10;
```

Comment:
Remove `|| 10`?.. Since perPage has already been found to be truthy `10` will never be used?